src/OrbitGl/Track.cpp

```diff
@@ -157,15 +167,15 @@ void Track::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset) {
   }
 
   // Draw collapsing triangle.
-  float button_offset = layout_->GetCollapseButtonOffset();
-  float toggle_y_pos = pos_[1] - half_label_height;
-  Vec2 toggle_pos = Vec2(tab_x0 + button_offset, toggle_y_pos);
-  collapse_toggle_->SetPos(toggle_pos[0], toggle_pos[1]);
-  collapse_toggle_->Draw(canvas, picking_mode, z_offset);
+  const float toggle_y_pos = DrawCollapsingTriangle(canvas, picking_mode, z_offset);
 
   // Draw label.
   if (!picking) {
     uint32_t font_size = layout_->CalculateZoomedFontSize();
+    // For the first 10 indentations, we decrease the font_size by 10 percent points (per
+    // indentation).
+    uint32_t capped_intent = (indentation_level_ > 9) ? 10 : indentation_level_;
+    font_size = (font_size * (10 - capped_intent)) / 10;
```

Comment:
So, starting at indentation level 10, the font size will be 0. Is this intended? And even at earlier indentation levels, the font size will be really small. 

I'm also wondering: We are only using indentation level 0 and 1 right now (correct?). Should we just remove any logic that deals with higher indentation levels?

Reply:
Agree. I will adjust this to be capped at 5 indentations (so the same as for the triangle toggle). I would still argue that it makes sense to keep the code being flexible. It does not make the code more complicated and in the future we might want to have tracks with more indentation (e.g. for the pipeline stats).